alt_e2eshark/e2e_testing/backends.py

```diff
@@ -34,36 +34,17 @@ class SimpleIREEBackend(BackendBase):
     '''This backend uses iree to compile and run MLIR modules for a specified hal_target_backend'''
     def __init__(self, *, device="local-task", hal_target_backend="llvm-cpu", extra_args : List[str] = None):
         self.device = device
+        if hal_target_backend == "hip":
+            print("IREE compiler python bindings do not currently support the change to using iree-hal-target-device=hip. Defaulting to depreciated iree-hal-target-backends=rocm")
+            hal_target_backend = "rocm"
```

Comment:
Got more context for this? Something doesn't seem right here...

also typo: "deprecated" not "depreciated"

Reply:
I'm looking at CompilerOptions and, for example, <https://github.com/iree-org/iree/blob/35dafeeb73ad7ace5b81b6250c3c29f6809ee2b4/compiler/bindings/python/iree/compiler/tools/core.py#L187>. This still uses `--iree-hal-target-backends` rather than `-iree-hal-target-device`. 

Will change the typo, thanks.